recipes/metrolopy/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "metrolopy" %}
+{% set version = "0.5.6" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 7802678a05b640bd91f32f33e3684d0edfea9e14c737db47de94c17898581109
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - numpy
```

Comment:
If it's pure Python, you shouldn't need numpy as a host dependency.

Reply:
Right, I have removed it from host (and also removed setup_requires numpy from setup.py in the project).